packages/ui/src/services/api/canUseEndpoint.test.ts

```diff
@@ -0,0 +1,77 @@
+import { ApiEndpoints } from "./canUseEndpoint"
+
+const mockUseApiModule = (useApi: boolean, useApiCaseEndpoint: boolean, useApiCaseIndexEndpoint: boolean) => {
+  jest.doMock("../../config.ts", () => ({
+    USE_API: useApi,
+    USE_API_CASE_ENDPOINT: useApiCaseEndpoint,
+    USE_API_CASES_INDEX_ENDPOINT: useApiCaseIndexEndpoint
+  }))
+}
+
+describe("canUseEndpoint", () => {
+  beforeEach(() => {
+    jest.resetModules()
+  })
+
+  afterEach(() => {
+    jest.resetModules()
+  })
+
+  it("returns false when USE_API is disabled", () => {
+    mockUseApiModule(false, true, false)
```

Comment:
```suggestion
  it("returns false when USE_API is disabled", () => {
    const useApi = false
    mockUseApiModule(useApi, true, false)
```

Something like this would improve readability of these tests and help remove cognitive load on the differences between each tests as it's hard to tell from the `mockUseApiModule` function call as all the arguments are booleans.

Reply:
👍 I'll do it once it's merged 🙂